mission-faq.liquid

```diff
@@ -0,0 +1,60 @@
+---
+layout: "default"
+title: Mystery Code Society | Mission FAQ
+---
+
+<section id="mission">
+  <p>Our Mission: To champion gender equity in tech through beginner and intermediate coding education for women, femmes, and non-men</p>
+</section>
+
+<section class="main">
```

Comment:
We overwrote the css to push down the section.main on this page in the mission-faq.css, and since we're now including all css for the entire app, there is a bug because of the loading order.

Therefore, this page is hidden beneath the mission banner.

Could we either add an additional class for specificity so the css applies, or maybe find some other solution?

<img width="1573" alt="Screen Shot 2020-03-01 at 4 08 32 PM" src="https://user-images.githubusercontent.com/11672661/75635026-6859ca80-5bd8-11ea-8fd4-39357183e09e.png">

Reply:
@Gmfholley I made the executive decision to move the content down on the homepage too, but let me know if you don't like that and I'll switch it back!